docs/sources/administration/manage-users-and-permissions/manage-server-users/view-user-account-details.md

```diff
@@ -0,0 +1,45 @@
++++
```

Comment:
View user account details, edit a user account, and view user sessions are so similar that they should be on the same page.

Reply:
Done.